Lombiq.NodeJs.Extensions/Docs/Markdown.md

```diff
@@ -0,0 +1,14 @@
+﻿# Scripts for MD files
+
+Since Markdown files don't need to be built, linting isn't included in the normal workflow but a separate target launches it for any project that uses _Lombiq.NodeJs.Extensions_. You can configure its behavior with the following MSBuild property:
+
+- `<MarkdownAnalysis>`:
+  - If unset, empty or "true", it engages the default behavior mentioned above.
```

Comment:
I'm not sure, I'd keep this as opt-in. So, be able to run it for the whole solution easily like now, but don't run it by default.

Reply:
Why? I left it enabled by default becuase linting just the files in the project is really fast.

Reply:
In our projects yes, but let's hope people don't keep a book or many OSS projects with documentation in their solution. Hooking into the build is invasive, and this project does multiple things, so it's not sure that everyone really wants Markdown linting too.